source/includes/_embed2.md.erb

```diff
@@ -165,12 +165,11 @@ Embed1 Feature | Status | Comments
 Additionally, all actions have been modified to return Promises.
 
 ## API Reference
```

Comment:
Its possible that this should be called `Settings` instead of `API Reference`. If that is the case we need to update the link on line 58